setup/github-bot/resources/github-bot.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=github-bot
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+ExecStart=/usr/bin/npm start
```

Comment:
I tend to avoid starting node services using `npm start` in production and using `node server.js` instead because npm heavily (and some times indeterministic) changes environment variables depending on what user it is run under. Not sure if this will be a problem for the bot though.